### PR TITLE
Moves harvest processing to fetch from MongoDB

### PR DESCRIPTION
**Pull request by benjwadams**
Updated to fetches and process harvests from MongoDB instead of PostgreSQL.

### CODE REVIEW

**Comment by lukecampbell** [comment]
What about the docker file?

**Comment by benjwadams** [comment]
Oops.  Let me get right on that.

File: Dockerfile [hunk 1]
```diff
@@ -9,11 +9,10 @@ RUN apt-get update && apt-get install -y \
       libxslt1-dev \
       libpq-dev \
       python-dev \
-      postgresql-client \
       python-pip
 RUN rm -rf /var/lib/apt/lists/*
 RUN pip install -U pip
-RUN pip install -e git+https://github.com/ioos/catalog-harvesting.git@28dc5d27e1b4939ce72c211b58e46f51b521f740#egg=catalog-harvesting
+RUN pip install -e git+https://github.com/benjwadams/catalog-harvesting.git@read_harvests_from_mongo#egg=catalog-harvesting
```
Comment (lukecampbell) [inline Dockerfile:15]: We probably don't want ur repo here

Comment (benjwadams) [inline Dockerfile:15]: yepp

File: Dockerfile [hunk 2]
```diff
@@ -9,11 +9,10 @@ RUN apt-get update && apt-get install -y \
...
 RUN useradd -m harvest
 RUN mkdir /var/log/harvest
```
Comment (benjwadams) [inline Dockerfile:17]: Probably want to move logs to a volume or have Docker handle them via other means?

Comment (lukecampbell) [inline Dockerfile:17]: I actually see a lot of docker images put it in `/var/log/<name>` but use a volume here. There's some other docker fanciness out in the world that can be used to pipe logs into the docker stdout too.

**Pull request merged**